network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/extensions/InjectionObservabilityAdderImpl.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.network.store.iidm.impl.extensions;
+
+import com.powsybl.commons.extensions.AbstractExtensionAdder;
+import com.powsybl.iidm.network.Injection;
+import com.powsybl.iidm.network.extensions.InjectionObservability;
+import com.powsybl.iidm.network.extensions.InjectionObservabilityAdder;
+import com.powsybl.network.store.iidm.impl.AbstractInjectionImpl;
+import com.powsybl.network.store.model.InjectionObservabilityAttributes;
+import com.powsybl.network.store.model.ObservabilityQualityAttributes;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+public class InjectionObservabilityAdderImpl<I extends Injection<I>>
+        extends AbstractExtensionAdder<I, InjectionObservability<I>>
+        implements InjectionObservabilityAdder<I> {
+
+    private boolean observable;
```

Comment:
```suggestion
    private boolean observable = false;
```
same, to see directly default values